extensions/amp-story/1.0/amp-story-desktop.css

```diff
@@ -82,10 +82,10 @@ amp-story[standalone][desktop] {
 }
 
 [desktop] amp-story-page {
-  transform: scale(1.0) translateX(-350%) translateY(0%) !important;
-  opacity: .05!important;
-  transform-origin: left!important;
-  border-radius: 16px!important;
+  transform: scale(1.0) translateX(250vw) translateY(0%) !important;
```

Comment:
Nit: here and below, we should stick to % over vw (see #15318)

Reply:
done.